src/main/java/org/dependencytrack/persistence/FindingsSearchQueryManager.java

```diff
@@ -0,0 +1,372 @@
+                queryFilter.append("TO_TIMESTAMP(:").append(paramName).append(", 'YYYY-MM-DD HH24:MI:SS')");
+            } else {
+                queryFilter.append(column).append(fromValue ? " >= :" : " <= :").append(paramName);
+            }
+            params.put(paramName, filter + (fromValue ? " 00:00:00" : " 23:59:59"));
+            if (!isMin) {
+                queryFilter.append(")");
+            }
+        }
+    }
+
+    private void processInputFilter(StringBuilder queryFilter, Map<String, Object> params, String paramName, String filter, String input) {
+        if (filter != null && !filter.isEmpty() && input != null && !input.isEmpty()) {
+            if (queryFilter.isEmpty()) {
+                queryFilter.append(" WHERE (");
+            } else {
+                queryFilter.append(" AND (");
+            }
+            String[] filters = filter.split(",");
+            for (int i = 0, length = filters.length; i < length; i++) {
+                switch (filters[i].toUpperCase()) {
+                    case "VULNERABILITY_ID" -> queryFilter.append("\"VULNERABILITY\".\"VULNID\"");
+                    case "VULNERABILITY_TITLE" -> queryFilter.append("\"VULNERABILITY\".\"TITLE\"");
+                    case "COMPONENT_NAME" -> queryFilter.append("\"COMPONENT\".\"NAME\"");
+                    case "COMPONENT_VERSION" -> queryFilter.append("\"COMPONENT\".\"VERSION\"");
+                    case "PROJECT_NAME" ->
+                            queryFilter.append("concat(\"PROJECT\".\"NAME\", ' ', \"PROJECT\".\"VERSION\")");
+                }
+                queryFilter.append(" LIKE :").append(paramName);
+                if (i < length - 1) {
+                    queryFilter.append(" OR ");
+                }
+            }
+            if (filters.length > 0) {
+                params.put(paramName, "%" + input + "%");
+            }
+            queryFilter.append(")");
+        }
+    }
+
+    private void preprocessACLs(StringBuilder queryFilter, final Map<String, Object> params) {
+        if (!isEnabled(ConfigPropertyConstants.ACCESS_MANAGEMENT_ACL_ENABLED)
+                || hasAccessManagementPermission(this.principal)) {
+            return;
+        }
+
+        if (queryFilter.isEmpty()) {
+            queryFilter.append(" WHERE ");
+        } else {
+            queryFilter.append(" AND ");
+        }
+
+        final var teamIds = new ArrayList<>(getTeamIds(principal));
+        if (teamIds.isEmpty()) {
+            queryFilter.append(":false");
+            params.put("false", false);
+            return;
+        }
+
+        queryFilter.append("""
+                EXISTS (
+                  SELECT 1
+                    FROM "PROJECT_ACCESS_TEAMS"
+                   WHERE "PROJECT_ACCESS_TEAMS"."PROJECT_ID" = "PROJECT"."ID"
+                     AND "PROJECT_ACCESS_TEAMS"."TEAM_ID" = ANY(%s)
+                )""".formatted(teamIds));
```

Comment:
`teamIds` should be passed as query parameters:

```suggestion
                     AND "PROJECT_ACCESS_TEAMS"."TEAM_ID" = ANY(:teamIds)
                )""");
        params.put("teamIds", teamIds);
```